WcaOnRails/spec/mailers/previews/wca_monthly_digest_mailer_preview.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/wca_monthly_digest_mailer
+class WcaMonthlyDigestMailerPreview < ActionMailer::Preview
+  def send_weat_digest_content
+    mail = nil
+    ActiveRecord::Base.transaction do
+      mail = WcaMonthlyDigestMailer.send_weat_digest_content
+      raise ActiveRecord::Rollback
```

Comment:
Why are you actively raising a rollback here? Was this c/p'ed from somehwere, and if yes, are you fully aware of why you c/p'ed it?